### PR TITLE
[WIP] Add support for COMMAND INFO command

### PR DESCRIPTION
**Pull request by sanimesh96**
Issue : [#148 ](https://github.com/DiceDB/dice/issues/148)

### CODE REVIEW

**Comment by sanimesh96** [comment]
Hi @JyotinderSingh 
Needed your help with something.
The implementation of `COMMAND INFO` mentioned in  Redis  command requires the presence of a commandTable which inclues all the necessary information. 
I am not sure if any file/commandTable exists which has all information about the commands. Can you point me in the right direction, if no such file/defination exits then do i need to create a file for the same?

**Comment by sanimesh96** [comment]
I have updated the PR @JyotinderSingh, Please have a look.

**Comment by AshwinKul28** [comment]
@sanimesh96 Please pull the latest from master which has linter version 1.60.1 that will fix the linter issue. Thanks

**Comment by AshwinKul28** [comment]
@sanimesh96 Any latest updates on this? Please pull the latest from the master, The changes looks good to me, just add unit tests for it.

File: core/eval.go [hunk 1]
```diff
@@ -2018,3 +2021,24 @@ func evalTOUCH(args []string) []byte {
 
 	return Encode(count, false)
 }
+
+// returns detailed information about the specified commands in the args
```
Comment (AshwinKul28) [inline core/eval.go:2025]:
```suggestion
// evalCommandInfo - returns detailed information about the specified commands in the args
```

**Comment by sanimesh96** [comment]
@AshwinKul28 i am working on adding the tests. Will update shortly

**Comment by AshwinKul28** [comment]
HI @sanimesh96 hope you are doing well. Any further updates on the tests you're adding last week?

**Comment by sanimesh96** [comment]
Hi @AshwinKul28, had a few concerns about this PR. This implementation will only work in the same way, if the metadata for commands are defined in the `diceCmds` defined in `commads.go`. But i dont see consistency across all the commands. Some have the attributes, some dont. how do you want me handling those cases?

**Comment by lucifercr07** [comment]
@sanimesh96 can you please rebase the PR once with latest changes.

**Comment by JyotinderSingh** [comment]
> Hi @AshwinKul28, had a few concerns about this PR. This implementation will only work in the same way, if the metadata for commands are defined in the `diceCmds` defined in `commads.go`. But i dont see consistency across all the commands. Some have the attributes, some dont. how do you want me handling those cases? Once clarified, will add the tests accordingly. Was getting some lint errors previously. Please create a build to see if it is fixed.

Please add any missing information in the commands.go file as a part of this effort. Additionally, ensure that the eval logic handles any missing values correctly.

**Comment by AshwinKul28** [comment]
@sanimesh96 Hello, Can you please resolve the conflicts and rebase the PR? Also, can we come to the closure of this PR soon, because a lot of changes are getting merged into the master and it will be difficult for you to rebase every time with the master. If you still have any concerns lets discuss them on the discord and close this out. Thanks so much for the work :D

**Comment by JyotinderSingh** [comment]
Closing this due to inactivity. Please let me know if you'd like me to reopen this.

**Pull request closed without merging**